seatunnel-connectors-v2/connector-clickhouse/src/main/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/sink/file/ClickhouseFileSinkWriter.java

```diff
@@ -58,24 +55,33 @@
 import java.util.stream.Collectors;
 
 @Slf4j
-public class ClickhouseFileSinkWriter implements SinkWriter<SeaTunnelRow, CKCommitInfo, ClickhouseSinkState> {
-    private static final String CLICKHOUSE_LOCAL_FILE_PREFIX = "/tmp/clickhouse-local/seatunnel-file";
+public class ClickhouseFileSinkWriter implements SinkWriter<SeaTunnelRow, CKFileCommitInfo, ClickhouseSinkState> {
+    private static final String CLICKHOUSE_LOCAL_FILE_PREFIX = "/tmp/seatunnel/clickhouse-local/seatunnel-file";
```

Comment:
Maybe we can set  `CLICKHOUSE_LOCAL_FILE_PREFIX` as a filed that can be set by configuration file. In some case, `/tmp` may be no big enough to save all temporary files.

Reply:
I will add option for user to config temp prefix value.